### PR TITLE
Fix a format string warning in the C++ examples

### PR DESCRIPTION
**Pull request by derobins**

### CODE REVIEW

**Comment by derobins** [comment]
Shows up in HPC CDash output

File: hl/c++/examples/ptExampleFL.cpp [hunk 1]
```diff
@@ -72,7 +72,7 @@ main(void)
     if (err < 0)
         fprintf(stderr, "Error getting packet count.");
 
-    printf("Number of packets in packet table after five appends: %llu\n", count);
+    printf("Number of packets in packet table after five appends: %" PRIdHSIZE "\n", count);
```
Comment (derobins) [inline hl/c++/examples/ptExampleFL.cpp]: count is hsize_t and should use the format specifier in H5public.h

Comment (jhendersonHDF) [reply hl/c++/examples/ptExampleFL.cpp]: Should this be PRIuHSIZE instead?

Comment (derobins) [reply hl/c++/examples/ptExampleFL.cpp]: Yes. Good catch!

**Review by jhendersonHDF — APPROVED** [review]
We should merge this after https://github.com/HDFGroup/hdf5/actions/runs/6646446878/job/18059890906 finishes to see if the next push to develop picks up the openmpi cache

**Review by lrknox — APPROVED** [review]

**Pull request merged**